### PR TITLE
docs: Event streaming fixes

### PR DESCRIPTION
**Pull request by ruchamahabal**
docs for recent changes in https://github.com/frappe/frappe/pull/11458

### CODE REVIEW

**Comment by ruchamahabal** [comment]
Spell check failing due to `eval`

File: erpnext_documentation/www/docs/user/manual/en/automation/event_streaming.md [hunk 1]
```diff
@@ -11,24 +11,40 @@ To access Event Streaming, go to:
...
 
 ## 2. How to set up Event Streaming
 
-### 2.1 Create an Event Producer
+Let's take 2 sites for explaining the process. http://test_site:8000 (Consumer site) and http://test_site_producer:8000 (Producer site)
+
+### 2.1 Obtain the Event Subscriber's keys from the producer site
+
+1. On http://test_site_producer:8000 (producer site), go to the User list.
+2. Open the user document you are going to use as an event subscriber. Scroll down to the section labelled "API access". In that section, generate keys for the user by clicking on **Generate Keys** button. You will get a prompt with the user secret, copy the user secret and save it with you. It will also generate an API key.
```
Comment (michellealva) [inline erpnext_documentation/www/docs/user/manual/en/automation/event_streaming.md:23]: "API Access"

File: erpnext_documentation/www/docs/user/manual/en/automation/event_streaming.md [hunk 2]
```diff
@@ -11,24 +11,40 @@ To access Event Streaming, go to:
...
+
+### 2.2 Generate Keys for the Event Subscriber on the consumer site
```
Comment (michellealva) [inline erpnext_documentation/www/docs/user/manual/en/automation/event_streaming.md:25]: Consumer Site

File: erpnext_documentation/www/docs/user/manual/en/automation/event_streaming.md [hunk 3]
```diff
@@ -11,24 +11,40 @@ To access Event Streaming, go to:
...
+
+1. On http://test_site:8000 (consumer site), go to the User list and follow the same process specified in the previous step.
+
+### 2.3 Create an Event Producer on the consumer site
```
Comment (michellealva) [inline erpnext_documentation/www/docs/user/manual/en/automation/event_streaming.md:29]: Event Producer on the Consumer Site

File: erpnext_documentation/www/docs/user/manual/en/automation/event_streaming.md [hunk 4]
```diff
@@ -11,24 +11,40 @@ To access Event Streaming, go to:
...
+
 1. The site which you want to subscribe to, is called as the Event Producer. Create an Event Producer document for the site you wish to get the updates from.
-2. Go to: **Home > Automation > Event Streaming > Event Producer**.
-3. Enter the site URL of the site you want to subscribe to, in the Producer URL field.
-4. Add all the Document Types you want to subscribe to, in the Event Configuration child table.
+2. On http://test_site:8000 (consumer site), go to **Home > Automation > Event Streaming > Event Producer**.
+3. Enter the site URL of the site you want to subscribe to (in this case http://test_site_producer:8000), in the Producer URL field.
```
Comment (michellealva) [inline erpnext_documentation/www/docs/user/manual/en/automation/event_streaming.md:33]: Enter the URL of the site you want to subscribe to

File: erpnext_documentation/www/docs/user/manual/en/automation/event_streaming.md [hunk 5]
```diff
@@ -11,24 +11,40 @@ To access Event Streaming, go to:
...
+4. Add all the Document Types you want to subscribe to, in the Event Producer Document Types child table.
```
Comment (michellealva) [inline erpnext_documentation/www/docs/user/manual/en/automation/event_streaming.md:34]: child table.. remove child?

File: erpnext_documentation/www/docs/user/manual/en/automation/event_streaming.md [hunk 6]
```diff
@@ -11,24 +11,40 @@ To access Event Streaming, go to:
...
 5. If you want to have the created documents with the same name as it is on the remote Event Producer site, check the 'Use Same Name' checkbox in the child table against the required Document Type.
```
Comment (michellealva) [inline erpnext_documentation/www/docs/user/manual/en/automation/event_streaming.md:35]: again.. remove child? Just keep table

File: erpnext_documentation/www/docs/user/manual/en/automation/event_streaming.md [hunk 7]
```diff
@@ -11,24 +11,40 @@ To access Event Streaming, go to:
...
-6. Set the Event Subscriber field to a user that will be used to create the documents fetched from the Event Producer. You need to create the same user both ways, i.e on the Event Consumer as well as the Event Producer site before creating the Event Producer.
-7. Save.
-8. The API key and API secret are then set in the Event Producer document for Authentication.
+6. Set the Event Subscriber field to the user that will be used to create the documents fetched from the Event Producer. You need to create the same user both ways, i.e on the Event Consumer as well as the Event Producer site before creating the Event Producer.
```
Comment (michellealva) [inline erpnext_documentation/www/docs/user/manual/en/automation/event_streaming.md:36]: i.e.

File: erpnext_documentation/www/docs/user/manual/en/automation/event_streaming.md [hunk 8]
```diff
@@ -11,24 +11,40 @@ To access Event Streaming, go to:
...
+7. Paste the API key and secret you generated in the first step (2.1) in the API key and API secret fields respectively.
```
Comment (michellealva) [inline erpnext_documentation/www/docs/user/manual/en/automation/event_streaming.md:37]: API Key and API Secret

File: erpnext_documentation/www/docs/user/manual/en/automation/event_streaming.md [hunk 9]
```diff
@@ -11,24 +11,40 @@ To access Event Streaming, go to:
...
+8. Save.
+9. After saving, an Event Consumer is created on the producer site (http://test_site_producer:8000). The keys of the user on the consumer site are automatically copied to the Event Consumer document on the producer site in this process.
 
     ![Event Producer](/docs/assets/img/automation/event-producer-doc.png)
 
-### 2.2 Approve Event Consumer on the Event Producer site
-1. After the Event Producer has been created, an Event Consumer automatically gets created on the other site. By default all the Subscribed Document Types have the status as 'Pending'. In order to enable the Event Consumer to consume the documents of these Document Types, their Status needs to be updated to 'Approved'.
+>**Note**: If at all the API Secret is changed for the users on any of these sites, you will have to manually update the keys in Event Producer as well as the Event Consumer on both the sites.
```
Comment (michellealva) [inline erpnext_documentation/www/docs/user/manual/en/automation/event_streaming.md:43]: update the keys in the Event Producer

File: erpnext_documentation/www/docs/user/manual/en/automation/event_streaming.md [hunk 10]
```diff
@@ -11,24 +11,40 @@ To access Event Streaming, go to:
...
+
+### 2.4 Approve Event Consumer on the Event Producer site
+
+1. After the Event Producer has been created, an Event Consumer automatically gets created on the producer site. By default all the Subscribed Document Types have the status as 'Pending'. In order to enable the Event Consumer to consume the documents of these Document Types, their Status needs to be updated to 'Approved'.
```
Comment (michellealva) [inline erpnext_documentation/www/docs/user/manual/en/automation/event_streaming.md:47]: By default, all the Subscribed ...

File: erpnext_documentation/www/docs/user/manual/en/automation/event_streaming.md [hunk 11]
```diff
@@ -48,7 +64,14 @@ If you have some places where internet connectivity is low, for example, a store
 
 ## 3. Features
 
-### 3.1 Event Update Log
+
+### 3.1 Unsubscribe from the updates
+
+As an event consumer, if you wish to unsubscribe from the updates for any doctypes you had previously subscribed to, check unsubscribe against the doctype. You will not receive any more updates from the producer site for that particular doctype once you have unsubscribed.
```
Comment (michellealva) [inline erpnext_documentation/www/docs/user/manual/en/automation/event_streaming.md:70]: Event Consumer

File: erpnext_documentation/www/docs/user/manual/en/automation/event_streaming.md [hunk 12]
```diff
@@ -48,7 +64,14 @@ If you have some places where internet connectivity is low, for example, a store
...
+
+![Unsubscribe](/docs/assets/img/automation/unsubscribe-event.png)
+
+### 3.2 Event Update Log
 Event Update Log logs every create, update, and delete for documents that have consumers on the Event Producer site.
```
Comment (michellealva) [inline erpnext_documentation/www/docs/user/manual/en/automation/event_streaming.md:75]: "Event Update Log" logs every create, update, and delete action for documents that have consumers on the Event Producer site.

**Review by michellealva — CHANGES_REQUESTED** [review]

**Review by michellealva — APPROVED** [review]

**Pull request merged**